src/preferences/dialog/dlgprefrecord.cpp

```diff
@@ -383,13 +384,32 @@ void DlgPrefRecord::loadMetaData() {
 
 void DlgPrefRecord::saveRecordingFolder()
 {
```

Comment:
```suggestion
void DlgPrefRecord::saveRecordingFolder() {
```

Reply:
puuh, it's like this in some places in this file..

Reply:
This is a bug fix PR. I am not doing a mass reformatting.

Reply:
sure.
I was under the impression we fix small style issues when touching a file, like QString > QStringLiteral.
604e317502
pick it if you like, or leave it

Reply:
rest LGTM and works as expected

Reply:
I just don't even want to talk about formatting and consider such discussions a waste of time.